src/main/resources/data/minebound/recipes/energized_iron_shovel.json

```diff
@@ -0,0 +1,19 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    " # ",
```

Comment:
shovels shouldn't have spaces so they can made in any of the three columns

Reply:
it works even with spaces. Ill change it anyway though